vuepress/docs/next/tutorials/create/mfe/angular.md

```diff
@@ -207,43 +207,38 @@ You should see a reponse after a few moments:
 You now have an Angular micro frontend running.
 :::
 
-## Build the Bundle and Install to Entando
+## Build, Publish and Install the Bundle
 
 ### Build and Publish the Bundle 
 
-1. Open `entando.json` in the bundle root folder and add the following to the `angular-widget` parameters. 
+1. Open `entando.json` in the bundle root folder and add the following under the `angular-widget` parameters. 
 ```json
 "buildFolder": "dist/angular-widget"
-
 ```
-2. Add a thumbnail JPEG or PNG file to the root bundle folder. It must be named thumbnail and not exceed 100kB, e.g. thumbnail.png
+2. (Optional) Add a thumbnail JPEG or PNG file to the root bundle folder. It must be named `thumbnail` and not exceed 100kB, e.g. thumbnail.png.
 
-3. Build the components
-```sh
-ent bundle build --all
-```
-4. Generate the Docker image for the bundle 
+3. Pack the bundle which includes the build and generate Docker images steps:
 ```sh
 ent bundle pack
 ```
-5. Publish the image
+4. Publish the image to a Docker registry:
 ```sh
 ent bundle publish 
 ```
-6. Deploy the bundle to the Hub
+5. Deploy the bundle to the Local Hub of your Entando Application:
 ```sh
 ent bundle deploy 
 ```
-7. Log into the App Builder and navigate to the Hub. Click `DEPLOY` for `YOUR-BUNDLE`. Click `install` in the pop-up window. 
+6. Log into the App Builder and navigate to the `Hub`. Click `DEPLOYED` for `YOUR-BUNDLE`. Click `Install` in the pop-up window. 
```

Comment:
the wording here might be a bit confusing because it comes on the heels of step 5., where the location of the bundle is stated to be the Local Hub. (i.e. why are we now calling this the `Hub`?) to clarify the intent you could say "Log into the App Builder and go to `Hub` in the left nav" or "Log into the App Builder and navigate to `Hub` in the left menu"

Reply:
I think sometimes it's implied and as long as it's referring to the same thing, it doesn't have to be local hub every single time.